### DISCUSSION TITLE
对于ios移动端中不使用播放器直接在alist中播放阿里云盘的视频有没有折中或蠢办法？

### DISCUSSION THREAD

**Discussion by mangic** [AlistGo/alist#913] [Q&A]
对于ios移动端中不使用播放器直接在alist中播放阿里云盘的视频有没有折中或蠢办法？
ios无法直接播放视频与content-type相关

**Comment by sunnyadams** [comment]
装个nplay，然后挂载webdav不就可以了么？

**Comment by xhofe** [comment]
这是阿里云盘和Safari的问题 而不是alist的问题。你应该去阿里云盘或Safari那里反馈而不是这里。
